exercises/linked-list/example.js

```diff
@@ -21,18 +21,18 @@ LinkedList.prototype.push = function LinkedList_push(value) {
   }
 };
 
-LinkedList.prototype.unshift = function LinkedList_unshift(value) {
+LinkedList.prototype.unshift = function LinkedListUnshift(value) {
   this.push(value);
   this._front = this._front.prev;
 };
 
-LinkedList.prototype.pop = function LinkedList_pop() {
-  if (this._front === null) {return undefined;}
+LinkedList.prototype.pop = function LinkedListPop() {
+  if (this._front === null) {return null;}
   this._front = this._front.prev;
   return this.shift();
 };
 
-LinkedList.prototype.shift = function LinkedList_shift() {
+LinkedList.prototype.shift = function LinkedListShift() {
```

Comment:
Can you remove these function names (on the right of `=`)? Since the functions are named on the prototype (`pop`, `shift`, etc) the names on the right don't matter - better just leave an anonymous function there. Like this:

```js
LinkedList.prototype.shift = function() {
....
}
```

Reply:
Sure, I will do that.